Common/FileUtil.h

```diff
@@ -0,0 +1,23 @@
+﻿/* Copyright (C) 2015 Rainmeter Project Developers
+*
+* This Source Code Form is subject to the terms of the GNU General Public
+* License; either version 2 of the License, or (at your option) any later
+* version. If a copy of the GPL was not distributed with this file, You can
+* obtain one at <https://www.gnu.org/licenses/gpl-2.0.html>. */
+
+#ifndef RM_COMMON_FILEUTIL_H_
+#define RM_COMMON_FILEUTIL_H_
+
+#include <string>
+#include <memory>
+
+namespace FileUtil
+{
```

Comment:
Can you change this to `namespace FileUtil {` and remove the indentation for the namespace block?